go.mod

```diff
@@ -434,3 +433,5 @@ replace oras.land/oras-go => oras.land/oras-go v1.2.4-0.20230801060855-932dd06d3
 // testcontainers-go has a bug with versions v0.25.0 and v0.26.0
 // ref: https://github.com/testcontainers/testcontainers-go/issues/1782
 replace github.com/testcontainers/testcontainers-go => github.com/testcontainers/testcontainers-go v0.23.0
+
+replace github.com/aquasecurity/trivy-aws => /Users/simarpreetsingh/repos/trivy-aws
```

Comment:
This PR requires the changes [here](https://github.com/aquasecurity/trivy-aws/pull/95) to be merged.